src/components/clubdetail/ClubDetailInfo/ClubDetailInfo.tsx

```diff
@@ -0,0 +1,84 @@
+import DeleteRoundedIcon from '@mui/icons-material/DeleteRounded';
+
+import { EditableField } from '../../EditableField/EditableField';
+import { ClubType } from '../../../model/Club';
+import style from './ClubDetailInfo.module.scss';
+
+type ClubDetailInfoProps = {
+  clubDetail: ClubType;
+  isOwner: boolean;
+  isMember: boolean;
+  memberCount: number;
+  onUpdate: (data: Partial<ClubType>) => void;
+  onDelete: () => void;
+  onLeaveClub: () => void;
+};
+
+export const ClubDetailInfo: React.FC<ClubDetailInfoProps> = ({
+  clubDetail,
+  isOwner,
+  isMember,
+  memberCount,
+  onUpdate,
+  onDelete,
+  onLeaveClub,
+}) => {
+  if (isOwner) {
```

Comment:
If these branches would grow and diverge even more, we should separate them.

Keep it as is for now, it is fine.